src/main/java/pages/HomePage.java

```diff
@@ -0,0 +1,31 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.PageFactory;
+
+public class HomePage {
+    private WebDriver driver;
+    private By greetingMessage = By.cssSelector("body > div.page-wrapper > header > div.panel.wrapper > div > ul > li.greet.welcome > span");
+    private By createAccountButton = By.xpath("/html/body/div[2]/header/div[1]/div/ul/li[3]/a");
+    private By signInButton = By.cssSelector("body > div.page-wrapper > header > div.panel.wrapper > div > ul > li.authorization-link > a");
```

Comment:
final and we don't have other selectors for these? like id, class etc

Reply:
For the create account button, yes, I will change it, I will also look for signInButton again. For the greetingMessage, not really, it;'s not a control but rather a text displayed.